guides/6.cli.rst

```diff
@@ -248,7 +248,15 @@ Gherkin Filters
 ---------------
 
 If you want to run only part of your suite, or some scenarios, you
-can do it with name or tags filters:
+can do it with name or tags filters.
+
+Tag filters supports three logical operators:
+
+- Tags separated by commas will be combined using the ``OR`` operation
+- Tags separated by ``&&`` will be combined using the ``AND`` operation
+- Tags preceded by ``~`` will be excluded, combined using the ``NOT`` operation
```

Comment:
By `combined` I was referring to being able to do something like `--tags="~@api&&~@javascript"` would allow combining the NOT's.  But point noted, I'll take out the word on that line.